### DISCUSSION TITLE
启用隧道不使用我设置的端口

### DISCUSSION THREAD

**Discussion by yozorachan** [natfrp/wiki#434] [使用问题]
![QQ截图20240205190039](https://github.com/natfrp/wiki/assets/105808870/337442fb-3f93-4e6c-bf37-888de8e60d19)
![QQ截图20240205190215](https://github.com/natfrp/wiki/assets/105808870/b2dba706-b076-41ad-ae94-ceac337b293e)
明明我设置的端口是8211，但是它好像一直在尝试使用不同的端口去启动隧道
今天上午隧道运行过程中突然发生的事件，无论是重启，重建隧道，强制TLS都无法解决
请问该如何解决？

**Comment by fakeboboliu** [comment]
你好，这是你到节点无法连接
如果尝试更换穿透节点后仍无法连接的话，可能是您的网络存在问题

**Reply by yozorachan** [reply]
但是ping测试是正常的
![QQ截图20240205192858](https://github.com/natfrp/wiki/assets/105808870/62ffa28b-c3e8-4545-8b1b-291117d1f764)

**Reply by LTHPKBTE** [reply]
ping 通和 TCP 连接建立是两个概念，有可能是当前网络环境或运营商拦截了frp相关流量
